uController.SourceGenerator/CodeGenerator.cs

```diff
@@ -89,11 +88,11 @@ public string Generate()
             var ctors = _model.HandlerType.GetConstructors();
             if (ctors.Length > 1 || ctors[0].GetParameters().Length > 0)
             {
-                WriteLine($"private readonly {S(typeof(ObjectFactory))} _factory = {S(typeof(ActivatorUtilities))}.CreateFactory(typeof({S(_model.HandlerType)}), {S(typeof(Type))}.EmptyTypes);");
+                WriteLine($"private readonly {typeof(ObjectFactory)} _factory = {typeof(ActivatorUtilities)}.CreateFactory(typeof({S(_model.HandlerType)}), {typeof(Type)}.EmptyTypes);");
```

Comment:
I just noticed the `S()` was unnecessary for the types I removed it from. I left `S()` for user-provided types because I'm guessing it's still necessary for that (but I'm not sure).

Reply:
Add them all back 😄